### PR TITLE
Typos tweaks

### PR DESCRIPTION
**Pull request by isralduke**
This PR:

- changes 4 spaces or tabs to just 2 spaces for code standards within `.njk` files
- Fixes missed typos in content
- Cleans up missed content cleanups from a previous PR

### CODE REVIEW

File: docs/form-controls/switches.md [hunk 1]
```diff
@@ -15,15 +15,11 @@ eleventyNavigation:
 
 - Please read [Label Guidance](/form-controls/labels-guidance) first.
 - Use this when users can toggle a feature on or off.
-- Switches are custom-styled [Checkboxes](/forms/checkboxes). 
-
-_**Note:** Some Switches can be marked as 'required' (<span class="fas fa-asterisk text-danger"></span>) in the wireframe/ mock-up. Please refer to [Required Form Inputs & Controls](/form-controls/labels-guidance#required-form-inputs-%26-controls) for guidance._
+- Users epxect Switches to take immediate effect.
```
Comment (james-alt) [inline docs/form-controls/switches.md:18]:
```suggestion
- Users expect Switches to take immediate effect.
```

File: docs/_includes/page-navigation.njk [hunk 2]
```diff
@@ -22,15 +22,15 @@
 <hr class="major opacity-50 mb-0"/>
 <div class="container-fluid py-12">
   <div class="row justify-content-between">
-    <div class="col-6 col-md-3 col-lg-2">
+    <div class="col-6 col-md-3 col-lg-4 col-xl-3">
```
Comment (james-alt) [inline docs/_includes/page-navigation.njk:25]: Should this file be updated to match the previous/next action button markup?

File: docs/_layouts/human-sitemap.njk [hunk 3]
```diff
@@ -4,7 +4,7 @@ layout: default.njk
 {% include "page-title.njk" %}
 
 <div class="sitemap p-lg-16">
-  <div class="container">
+  <div class="container--fluid">
```
Comment (james-alt) [inline docs/_layouts/human-sitemap.njk:7]:
```suggestion
  <div class="container-fluid">
```

**Review by james-alt — CHANGES_REQUESTED** [review]
Some minor changes, otherwise this is pretty good to go. As to our other discussion I think we could remove any of the form templates where we are saying that someone should be using MyLa.

File: docs/_includes/markup/button-previous-next-page.njk [hunk 4]
```diff
@@ -1,14 +1,14 @@
-<div class="btns-previous-forward">
+<div class="btns-previous-next">
```
[the remark this answers is not in the document]
Comment (isralduke) [reply docs/_includes/markup/button-previous-next-page.njk:1]: On that i can go either way. Do we want to emphasize going forward over going to previous? in workflows, we might want to emphasize going forward. in documentation, that might not matter. @james-alt

File: docs/components/popovers-tooltips.md [hunk 5]
```diff
@@ -22,18 +22,32 @@ eleventyNavigation:
 
 ### Popover
 
-{% include 'markup/popovers.njk' %}
```
[the remark this answers is not in the document]
Comment (isralduke) [reply docs/components/popovers-tooltips.md:25]: Easier editing and they aren’t used anywhere else.

File: docs/components/data-tile.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ eleventyNavigation:
 
 ## Usage
 
-<div class="data-tile card shadow-none">	
+<div class="data-tile shadow-none">
```
[the remark this answers is not in the document]
Comment (isralduke) [reply docs/components/data-tile.md:24]: Easier editing and its not used anywhere else in the project

File: docs/_includes/page-navigation.njk (hunk 2 above)
Comment (isralduke) [reply docs/_includes/page-navigation.njk:25]: i could go either way on this. documentation may not need to emphasize going to the next screen. (but a sequential process might want to emphasize going to the next screen.) maybe leave this as is for now

**Review by isralduke — COMMENTED** [review]
made feedback

**Pull request closed without merging**